assets/src/js/commands/admin-commands.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * Admin Commands
+ *
+ * Core WordPress commands for Secure Custom Fields administration.
+ * This file registers navigation commands for all primary SCF admin screens,
+ * enabling quick access through the WordPress commands interface (Cmd+K / Ctrl+K).
+ *
+ * @since 6.5.0
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createElement } from '@wordpress/element';
+import { Icon } from '@wordpress/components';
+import { dispatch } from '@wordpress/data';
+import { createQueue } from '@wordpress/priority-queue';
+
+/**
+ * Initialize deferred execution queue and context
+ */
+const queue = createQueue();
+const context = {};
+
+/**
+ * Register admin commands for SCF
+ */
+queue.add( context, () => {
```

Comment:
It's not really clear to me why you're using "priority-queue" here? Did you actually notice performance issues or something. Also potentially you can just use `requestIdleCallback` directly instead of using a queue with a single callback.

Reply:
I did not notice any performance difference; I implemented the priority queue based on Greg's feedback [here](https://github.com/WordPress/secure-custom-fields/pull/121#discussion_r2060323077) to defer the loading.

Reply:
Let's just get rid of it. I think `defer` is enough.

Reply:
I simplified to a `requestIdleCallback` here before reading your answer. Should I get rid of that, too? It still seems overkill.